### PR TITLE
Added compatibility for Python3

### PR DESCRIPTION
**Pull request by Engerrs**

### CODE REVIEW

**Comment by Engerrs** [comment]
Hi @wardi , added a compatibility with Python 3.

File: ckanext/fluent/validators.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
 import json
 import re
+import six
 
-from ckan.plugins.toolkit import missing, _, get_validator, Invalid
-from pylons import config
+from ckan.plugins.toolkit import missing, get_validator, Invalid, check_ckan_version
+from ckan.common import config, _
```
Comment (smotornyuk) [inline ckanext/fluent/validators.py]: install `ckantoolkit` module and import `config` and `_` from it - there were no `config` inside `ckan.common` before 2.6

File: ckanext/fluent/validators.py [hunk 2]
```diff
@@ -81,7 +84,7 @@ def validator(key, data, errors, context):
         value = data[key]
         # 1 or 2. dict or JSON encoded string
         if value is not missing:
-            if isinstance(value, basestring):
+            if isinstance(value, str if ckan_python3 else six.string_types):
```
Comment (smotornyuk) [inline ckanext/fluent/validators.py]: `if isinstance(value, six.string_types):` - will do the same thing

File: ckanext/fluent/validators.py [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
+from ckan.plugins.toolkit import missing, get_validator, Invalid, check_ckan_version
+from ckan.common import config, _
 
 from ckanext.fluent.helpers import (
     fluent_form_languages, fluent_alternate_languages)
 from ckanext.scheming.helpers import scheming_language_text
 from ckanext.scheming.validation import (
     scheming_validator, validators_from_string)
 
+ckan_python3 = check_ckan_version('2.9')
```
Comment (smotornyuk) [inline ckanext/fluent/validators.py]: use `six.PY3` instead - it's more ideomatic

**Pull request merged**